docs/managed-datahub/observe/data-contract.md

```diff
@@ -0,0 +1,152 @@
+# Data Contracts
+
+## What Is a Data Contract
+
+The definition of a Data Contract is consisted of the following:
```

Comment:
we probably should have a high level sentence about what a data contract is, before diving into the properties